ci/test.sh

```diff
@@ -26,13 +31,17 @@ function main {
   [[ $# -eq 0 || "$1" == '-h' || "$1" == '--help' ]] && helpAndExit 0
   REL=`dirname "$0"`
   eval "$REL/prepare_db.sh '$1'"
-  if [[ "$1" == "H2" ]]; then
-    (cd $REL/../web && npm run test)
-    mvn clean verify -q -f $REL/../lib/ -B
-    mvn clean install -q -f $REL/../rest/ -B -P history.plugin
-  elif [[ "$1" == "POSTGRES_10_4" ]]; then
+  if [[ "$1" == "H2" && "$2" == "LIB" ]]; then
+    mvn clean install -q -f $REL/../lib/ -B
```

Comment:
is there a reason why we install the artifacts again?

Reply:
We have to do a complete installation with tests in this stage, the main stage is H2 && REST.
We can skip javadoc in this installation too(I will add it).